src/constructs/ecs/ecs-task.ts

```diff
@@ -227,9 +229,21 @@ export class GuEcsTask extends Construct {
         streamPrefix: app,
         logRetention: 14,
       }),
-      readonlyRootFilesystem,
+      readonlyRootFilesystem: true,
     });
 
+    if (volumeMountPath) {
+      taskDefinition.addVolume({
+        name: `${app}-volume`,
+      });
+
+      containerDefinition.addMountPoints({
+        sourceVolume: `${app}-volume`,
```

Comment:
Does this work only because the string matches `name` in `taskDefinition.addVolume` above? If so, I think we could make this more explicit via:

```ts
import type { Volume } from 'aws-cdk-lib/aws-ecs';

const volume: Volume = {
  name: `${app}-volume`,
};

taskDefinition.addVolume(volume);

containerDefinition.addMountPoints({
  sourceVolume: volume.name,
  containerPath: volumeMountPath,
  readOnly: false
});
```